src/components/AddView.js

```diff
@@ -1,10 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
 const AddView = () => {
+  const [state, setState] = useState({
```

Comment:
Maybe you could rename that piece of state into ```item```/```setItem```. Then ``purchaseItem``  could be called as ```value``` or ```type```?